recipe_scrapers/usapears.py

```diff
@@ -11,6 +11,18 @@ class USAPears(AbstractScraper):
     def host(cls):
         return "usapears.org"
 
+    def author(self):
+        author = self.schema.author()
+        if not author:
+            twitter_data = self.soup.find(
+                "meta", {"name": "twitter:data1", "content": True}
+            )
+            if twitter_data and self.soup.find(
+                "meta", {"name": "twitter:label1", "content": "Written by"}
+            ):
+                author = twitter_data["content"]
+        return author
```

Comment:
I'd like to suggest that we group the two retrieval approaches more closely, and reduce the indentation level.

Something like:

```suggestion
    def author(self):
        author = self.schema.author()
        if author:
            return author

        d1 = self.soup.find("meta", {"name": "twitter:data1", "content": True})
        l1 = self.soup.find("meta", {"name": "twitter:label1", "content": "Written by"})
        if d1 and l1:
            return d1["content"]
```